core/src/main/java/org/apache/calcite/jdbc/Driver.java

```diff
@@ -101,13 +101,12 @@ public CalcitePrepare createPrepare() {
     if (prepareFactory != null) {
       return prepareFactory.get();
     }
-    return createPrepareFactory().apply();
+    return CalcitePrepare.DEFAULT_FACTORY.apply();
   }
 
   /** Returns a factory with which to create a {@link CalcitePrepare}.
    *
-   * <p>Now deprecated; if you wish to use a custom prepare, overrides of this
-   * method will still work, but we prefer that you call
+   * <p>Now deprecated; if you wish to use a custom prepare, please call
```

Comment:
Can't we override this method now?

Reply:
Yes, we can override `createPrepare()` to achieve the same goal.
Since `createPrepareFactory()` is deprecated, maybe we shouldn't call or override it any more.